src/extensions/scratch3_vizblocks/index.js

```diff
@@ -562,6 +572,24 @@ class Scratch3VizBlocks {
                         description: 'draw line'
                     })
                 },
+                {
+                    opcode: 'drawBar',
```

Comment:
Change all to `drawBars`

```suggestion
                    opcode: 'drawBars',
```